src/lib/mina_graphql/mina_graphql.ml

```diff
@@ -346,6 +346,57 @@ module Types = struct
             ~resolve:(fun _ {Fee_transfer.fee; _} -> Currency.Fee.to_uint64 fee)
         ] )
 
+  let account_timing : (Mina_lib.t, Account_timing.t option) typ =
+    obj "AccountTiming" ~fields:(fun _ ->
+        [ field "timed_minimum_balance" ~typ:uint64
```

Comment:
actually, I think this can be `initial_minimum_balance`

Reply:
sounds good!